### PR TITLE
Miscellaneous edits

### PR DESCRIPTION
**Pull request by tsalo**
@vanandrew sorry for the unsolicited PR. I was reading through the repo and thought some stylistic changes would make it easier to understand for potential reviewers.

Changes proposed:

1. Run isort and black on the Python files.
2. Add `omni` to the requirements in pyproject.toml. I don't know if there's a specific version you used though.
    - I also sorted the requirements to make them easier to read.
3. Add badge for the preprint to the README.
4. Add text on the scope of the repo to the README. I think this needs to be further expanded.
5. Remove `*` imports in `medic_analysis.common.__init__`. I believe I've fixed any imports that rely on direct imports from `medic_analysis.common`.
6. Remove unused imports in Python scripts.
7. Start adding docstrings to modules and (eventually) functions.

### CODE REVIEW

File: medic_analysis/scripts/tSNR_processing.py [hunk 1]
```diff
@@ -83,7 +84,13 @@ def main():
...
                     futures.append(
                         executor.submit(
-                            compute_tSNR, run_dir, pipeline, subject_dir.name, session_name, run_dir.name, wmparc_path
+                            compute_tSNR,
+                            run_dir,
+                            pipeline,
+                            subject_dir.name,
+                            session_name,
+                            run_dir.name,
+                            wmparc_path,
```
Comment (tsalo) [inline medic_analysis/scripts/tSNR_processing.py:93]: This variable is undefined.

File: README.md [hunk 2]
```diff
@@ -21,20 +30,20 @@ To call them, simply type the name of the script without the `.py` extension.
 
 The two main scripts used in the paper are:
```
Comment (tsalo) [inline README.md:31]: What about the other scripts? Were they considered extraneous for the preprint?

**Comment by vanandrew** [comment]
Hey @tsalo. Thanks for doing this!

I'll have to go back and take a look at this codebase again to refresh myself before reviewing your PR, but I'll try to fit in some time to do so.

File: README.md (hunk 2 above)
Comment (vanandrew) [reply README.md:31]: If I recall correctly, I think some of these scripts were actually last minute analyses that my Thesis Committee wanted to see for my defense.

I might just remove these and put them in another repo.

Comment (tsalo) [reply README.md:31]: It might be enough to just put them in a new branch.

File: medic_analysis/scripts/tSNR_processing.py (hunk 1 above)
Comment (vanandrew) [reply medic_analysis/scripts/tSNR_processing.py:94]: Addressed in https://github.com/tsalo/medic_analysis/pull/1

File: README.md (hunk 2 above)
Comment (vanandrew) [reply README.md:31]: Unnecessary files are removed and additional comments added for other script files in https://github.com/tsalo/medic_analysis/pull/1

**Comment by vanandrew** [comment]
I created a PR for your PR here: https://github.com/tsalo/medic_analysis/pull/1

It mainly removes a few unnecessary files and adds some additional docs.

**Comment by vanandrew** [comment]
I can go ahead and merge this in if there aren't anymore edits.

**Comment by tsalo** [comment]
Thanks! I'll open a separate PR if I see anything else.

**Pull request merged**